src/i18n/index.js

```diff
@@ -1,13 +1,16 @@
 import enUS from "./en-us"
+import ru from "./ru"
 
 // ADD LANGUAGES HERE
 const languages = [
-    { name: "English", code: "en-us", flag: "us" }
+    { name: "English", code: "en-us", flag: "us" },
+    { name: "Russian", code: "ru", flag: "ru" }
 ]
 
 export { languages }
 
 // DO NOT MODIFY THIS EXPORT, LANGUAGE FILES CAN BE DYNAMICALLY LOADED
 export default {
-    "en-us": enUS
+    "en-us": enUS,
+    "ru": ru
```

Comment:
The language select should work without you exporting the `ru` here. We want to keep the exports to the minimum so that we don't load any languages we may not need.

Try and see if this works and if it doesn't then let me know and i'll look into it.

Reply:
Seems to be working without export